src/bulk-load.ts

```diff
@@ -48,6 +48,7 @@ interface InternalOptions {
   fireTriggers: boolean;
   keepNulls: boolean;
   lockTable: boolean;
+  order: {columnName: string, direction: 'ASC' | 'DESC'}[];
```

Comment:
I'm wondering if it would make sense to change this to an object, instead of an array of objects?

```suggestion
  order: { [columnName: string]:  'ASC' | 'DESC' };
```

It never makes sense to specify a column name twice in the order, and this would remove a bunch of intermediate objects. When iterating over object properties, they are ordered in the order that they are defined - this is guaranteed by the EcmaScript spec nowadays.

Using this would look something like this:

```js
connection.newBulkLoad('#tmpTestTable', { order: { 'nnn': 'ASC', 'sss': 'DESC' } }, ...
```

What do you think? 🙇‍♂️

Reply:
Your approach is much cleaner. I'm learning new things about this language every day. Thank you very much!